week-3/quote-generator/quotes.js

```diff
@@ -25,469 +25,509 @@ function pickFromArray(choices) {
 const quotes = [
   {
     quote: "Life isn't about getting and having, it's about giving and being.",
-    author: "Kevin Kruse",
+    author: "- Kevin Kruse",
```

Comment:
Instead of manually adding the dash ("-") to each quote in the array, you could use JS to insert it into your HTML code dynamically :) 

```
quoteParagahraph2.innerText = `- ${currentQuote.author}`;

```
By using JS to handle the quote and formatting, your code becomes more flexible and easier to maintain!